src/bar.cpp

```diff
@@ -305,6 +305,7 @@ void waybar::Bar::setupAltFormatKeyForModuleList(const char* module_list_name) {
 void waybar::Bar::handleSignal(int signal) {
   for (auto& module : modules_left_) {
     auto* custom = dynamic_cast<waybar::modules::Custom*>(module.get());
+    module->update();
```

Comment:
Why did you added that?

Reply:
Whoops, that snuck in from a patch I maintain locally! Removed it from the patch :+1: